### ISSUE TITLE
Training About

### ISSUE THREAD

**Issue by HLZExplorer** [hkchengrex/STCN#158]
First of all, thank you for your contribution and excellent work. 
Recently, I have been trying to replicate your work. I first pre-trained on static images and BL30K, then main train on DAVIS and YouTubeVOS, and finally used the trained weights for inference. However, when I uploaded the inference results for 2017testdev and 2018YouTubeVOS, the metrics were much lower than yours (about 10 points lower). Just to clarify, I trained on two 3080TI GPUs, and I've ensured that the hyperparameters are exactly as you specified. However, the inference results suggest that I may have overlooked some details. Could you please advise on what might be causing this discrepancy? Also, I noticed in your paper that you mentioned freezing batch normalization layers during main training. Do I need to explicitly freeze them, as I haven't seen any freezing operations in the code?
![1713432748671](https://github.com/hkchengrex/STCN/assets/167385475/a3f9ebe0-ec0b-4744-b464-55f69cbe79d8)

**Comment by hkchengrex** [comment]
The default setting, without any modification, should work. See https://github.com/hkchengrex/STCN/issues/108

**Comment by HLZExplorer** [comment]
I'll check the training details again and thank you for your reply
